src/accountImporter.ts

```diff
@@ -258,65 +250,65 @@ var _validateRequiredParameters = function (options) {
     default:
       throw new FirebaseError("Unsupported hash algorithm " + clc.bold(options.hashAlgo));
   }
-};
+}
 
-var _validateProviderUserInfo = function (providerUserInfo) {
+function validateProviderUserInfo(providerUserInfo: any): any {
```

Comment:
This can return `{ error?: string }` instead of any

Reply:
Yes